src/actions/actions/actions.js

```diff
@@ -12,11 +12,13 @@ import { LIST_ACTIONS } from '@plone/volto/constants/ActionTypes';
  * @returns {Object} List actions action.
  */
 export function listActions(url) {
-  return {
-    type: LIST_ACTIONS,
-    request: {
-      op: 'get',
-      path: `${url}/@actions`,
-    },
+  return async function (dispatch, getState) {
```

Comment:
Why is this style of action needed? I don't know if this is a default feature of react-redux or it comes to us thanks to thunks support, but we're already able to treat the actions as async and wait their result, see for ex https://github.com/plone/volto/blob/de3cb700c2ba44e7b19c3ebafc98d2bbba0852e9/src/components/manage/Widgets/SelectAutoComplete.jsx#L150

Reply:
I think @sneridagh is the one to weigh in on this, no?